pkg/bisect/bisect.go

```diff
@@ -50,12 +50,14 @@ type ReproConfig struct {
 	C    []byte
 }
 
+// TODO: use better interface names?
```

Comment:
Yes, the `instance.EnvInterface` should be just `instance.Env`. Adding Interface to each interface is like adding Struct, Function and Variable suffixes to every struct, function and variable in C/C++.
Consequently you will need to rename the current `instance.Env` to something else (maybe just `env`, since it will become local to the package, naming requirements are softer).

Reply:
Thanks, I've made a name change. It seems like this interface is used inside syz-testbuild/testbuild.go:func(). Is there a better way?